vars/dockerExecute.groovy

```diff
@@ -134,7 +134,7 @@ void call(Map parameters = [:], body) {
 
         if (isKubernetes() && config.dockerImage) {
             List dockerEnvVars = []
-            config.dockerEnvVars.each { key, value ->
+            config?.dockerEnvVars?.each { key, value ->
```

Comment:
config is never null if `config.dockerImage` is true, so that null-safe operator is misleading